### PR TITLE
Use identifiers rather than string keys in wasmImports object.

### PR DESCRIPTION
**Pull request by sbc100**
We should get a lot more code savings from this change once we switch
from using the underscore name mangling on the JS side.   Once we do
that most of the imports will use the abbreviated the object syntax of
just `foo,` rather then `foo = _foo,`

### CODE REVIEW

File: emscripten.py [hunk 1]
```diff
@@ -719,8 +719,13 @@ def create_sending(metadata, library_symbols):
             continue
           send_items_map[demangled] = f
 
-  sorted_keys = sorted(send_items_map.keys())
-  return '{\n  ' + ',\n  '.join('"' + k + '": ' + send_items_map[k] for k in sorted_keys) + '\n}'
+  sorted_items = sorted(send_items_map.items())
+  prefix = ''
+  if settings.USE_CLOSURE_COMPILER:
+    # This prevents closure compiler from minifying the field
+    # name in this object.
```
Comment (kripken) [inline emscripten.py:726]:
```suggestion
    # names in this object.
```

File: test/optimizer/applyDCEGraphRemovals-output.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 var name;
 
 var wasmImports = {
- "save1": 1,
- "save2": 2
+ save1: 1,
+ save2: 2
```
Comment (kripken) [inline test/optimizer/applyDCEGraphRemovals-output.js:5]: Do we never have export names that need quoting for other reasons? For example, a name with a space in it, like this:

```cpp
__attribute__((import_module("module"),
               import_name("spaces in between"))) void func();
```

Comment (sbc100) [reply test/optimizer/applyDCEGraphRemovals-output.js:5]: I'm not sure that can work in our current code base, with or without this change.

If we want to support that kind of thing we should add a test for it.

Trying out the above today is currently not possible... even if I write a library js file like this:

```
mergeInto(LibraryManager.library, {
  "spaces in between": function() {
    console.log('XXX');
    return 0;
  },
});
```

Results in the following error at runtime:

```
$ node a.out.js 
/usr/local/google/home/sbc/dev/wasm/emscripten/a.out.js:1076
  function _spaces in between() {
                   ^^

SyntaxError: Unexpected token 'in'
```

So... this change isn't breaking anything that isn't already broken.

**Review by kripken — APPROVED** [review]
Sounds good, thanks for checking.

**Pull request merged**